### ISSUE TITLE
ingress.path not working

### ISSUE THREAD

**Issue by pikomen** [netdata/helmchart#130]
helm install \
--name=netdata \
--set ingress.path=/netdata  \
--set parent.database.persistence=false  \
--set parent.alarms.persistence=false \
--set service.type=LoadBalancer \
--namespace netdata \
./netdata-helmchart

k8s 1.16.10

logs from ingress:
"GET /netdata HTTP/1.1" 404 101 "-" "Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36" 520 0.001 [netdata-netdata-http] [] 10.233.65.20:19999 101 0.000 404 8484098d1dab671264b2601699df380d

**Comment by cakrit** [comment]
We should change the documentation of this option to state that when someone wants to use a different ingress path, they need a proxy in front of netdata, that will be configured to translate the path to / before sending the request to netdata itself.

**Comment by prologic** [comment]
> We should change the documentation of this option to state that when someone wants to use a different ingress path, they need a proxy in front of netdata, that will be configured to translate the path to / before sending the request to netdata itself.

I don't agree with this. It is very typical and normal for web applications to be configured with a `BaseURL` option. We _should_ do this, not making our users have to put a Reverse Proxy in front just to rewrite inbound/outbound paths. If we truely want to remain true to our "zero configuration" and "low touch" capabilities, just simply provide a single option to correct this behaviour is __far simpler__ than standing up a Reverse Proxy for some kind and figuring out how to rewrite ingress/egress request paths.

**Comment by pikomen** [comment]
@prologic you are reading my minds

**Comment by ncmans** [comment]
@prologic @cakrit I suggest we look at existing projects and see what passes for "expected" in k8s. We can then evaluate it and see if makes sense for our case.

**Comment by ncmans** [comment]
OK, I looked at a few examples (grafana, unifi, etc.). They all have the `ingress.path` as an option, but anything other than `/` won't work! I guess one project copied from another and here we are.

So I agree with @prologic that we should either:

- have an option to netdata to specify base path
- use relative path in the web app everywhere

**Comment by cakrit** [comment]
@ncmans no need to ask for my input in these, you own it and know what's best. Just please no matter what you do, maintain the zero configuration principle with the default settings. i.e. a user should not HAVE to modify any settings, in order to get netdata installed and working.
